src/web/RPCServerHandler.h

```diff
@@ -89,8 +89,8 @@ class RPCServerHandler
             auto req = boost::json::parse(request).as_object();
             LOG(perfLog_.debug()) << connection->tag() << "Adding to work queue";
 
-            if (not connection->upgraded and not req.contains("params"))
-                req["params"] = boost::json::array({boost::json::object{}});
+            if (not connection->upgraded && shouldReplaceParams(req))
```

Comment:
small thing : "and" is better here, because "not" had been used here.

Reply:
Agree!